Sources/PortalSwift/Core/Api/PortalApi.swift

```diff
@@ -168,20 +168,20 @@ public class PortalApi {
     throw URLError(.badURL)
   }
 
-  public func getNFTs(_ chainId: String) async throws -> [FetchedNFT] {
-    if let url = URL(string: "\(baseUrl)/api/v3/clients/me/nfts?chainId=\(chainId)") {
+  func getNftAssets(_ chainId: String) async throws -> [NftAsset] {
```

Comment:
waiting on kelson to see if the return type is different for solana vs eip155. Once he responds with yes we can approve and merge. if not we will have to make changes similar to the build transaction method.